<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2160</width>
    <height>3840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/image/kkk3.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>1821</width>
      <height>871</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/Add a heading.png);
background-color: rgba(0, 0, 0, 15);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mosque">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>1520</y>
      <width>1451</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_formal">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>2050</y>
      <width>1451</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_wedding">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>2580</y>
      <width>1451</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2160</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
